### PR TITLE
Logger logrus implement

### PR DESCRIPTION
**Pull request by printfcoder**

### CODE REVIEW

**Comment by k1ng440** [comment]
i believe uber/zap sugar logger is much faster than logrus and interfaces are pretty close

**Comment by vtolstov** [comment]
looks good

File: logger/logrus/log_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package logrus
+
+import (
+	"testing"
+
+	log "github.com/micro/go-micro/logger"
```
Comment (vtolstov) [inline logger/logrus/log_test.go]: log "github.com/micro/go-micro/logger" => log "github.com/micro/go-micro/v2/logger"

File: logger/logrus/logrus.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
+package logrus
+
+import (
+	"io"
+	"os"
+
+	log "github.com/micro/go-micro/logger"
```
Comment (vtolstov) [inline logger/logrus/logrus.go]: log "github.com/micro/go-micro/logger" => log "github.com/micro/go-micro/v2/logger"

File: logger/logrus/options.go [hunk 3]
```diff
@@ -0,0 +1,86 @@
+package logrus
+
+import (
+	"context"
+	"io"
+
+	log "github.com/micro/go-micro/logger"
```
Comment (vtolstov) [inline logger/logrus/options.go]: log "github.com/micro/go-micro/logger" => log "github.com/micro/go-micro/v2/logger"

**Comment by vtolstov** [comment]
also as we have go.mod per package you need to create go.mod for this logrus logger package

**Comment by vtolstov** [comment]
to avoid duplicate blocks
```
 if o.Context == nil {                   
   o.Context = context.Background()      
 }                                       
```

**Comment by printfcoder** [comment]
@vtolstov go mod is updated

File: logger/logrus/go.mod [hunk 4]
```diff
@@ -0,0 +1,8 @@
+module github.com/micro/go-plugins/logger/logrus
+
+go 1.13
+
+require (
+	github.com/micro/go-micro/v2 v2.0.2
```
Comment (vtolstov) [inline logger/logrus/go.mod]: where is tag for this version in go-micro repo? as logger merged not so far, i think that version 2.0.1-xxx (master branch commit with date)

**Pull request merged**